docs/tutorials/computers101.md

```diff
@@ -0,0 +1,225 @@
+
+**Keyboard Shortcuts**
+
+|<div style="width:150px">Shortcut</div> |<div style="width:300px">Description</div> | 
+|---------------------------------------|------------------------------------------|
+|<table><tr><td><kbd>**&uarr;**</kbd><kbd>**&darr;**</kbd></td></tr></table>| Use the arrow keys on your keyboard at the prompt to scroll through previous commands, one line at a time, instead of scrolling through the terminal with your mouse|
+|<kbd>Ctrl + A</kbd>| Use to navigate to the beginning of a line|
+|<kbd>Ctrl + E</kbd>| Use to navigate to the end of a line|
+|<table><tr><td><kbd>Tab</kbd></td></tr></table>|Use **Tab Completion** to automatically fill in the name of a file or directory instead of manually doing so <br> *Note: if mulitple file or directory names are named similarly in the beginning of their title, the tab completion key won't work until the name is unique*<br>Ex: If there are text files `MyNovelBeg.txt` `MyNovelMid.txt` and `MyNovelEnd.txt` and we want to type `MyNovelBegEnd.txt` tab completion will work until `MyNovelE` is typed and the **<kbd> Tab </kbd>**key is pressed
+|~|-the tilde represents the user's home directory<br>-can be used to navigate directories without typing out the user's home directory|
+|* or ?|**Wildcards**<br>-a character that can be used to represent a set of none or more characters<br>-can be used for ease of navigating files and directories in the shell|
+
+## Other Notes
+###Directories
+**Directory names** are written as: `DirectoryName/` with the slash mark to indicate it is a directory
+<br><br>**Recursive** - a directory can contain multiple files and/or multiple directories that each have multiple files and/or directories. If a command or program is "recursive," it will act on a directory and all of the contents of that directory
+###Scripts
+**Script** - a list of a particular sequence of commands, usually stored in a file or saved for future reference
+<br><br>**Leaving Comments**
+<br>&ensp;&ensp;-the # symbol followed by a string can be used to leave a comment in a script that will be skipped over by the shell, so that it won't run as a command
+<br>&ensp;&ensp;-this can be useful for leaving notes on what your command does for others or for future reference 
+
+## How to `git and github`- Using a Version Control System
+
+!!! note "Summary"
+    See <a href="https://swcarpentry.github.io/git-novice/">Software Carpentry</a> for instruction. Below are key take aways and what to look for in the link.
+    This guide will provide key concepts needed to navigate and work with `git` in order to make git push and pull requests. Defining what a version control system is, understanding 
+    basic git commands, initiating repositories, and utilizing `git add` and `git commit` are the crucial to successfully utilize `git` and `github`. 
+    <br>*Note: A basic understanding of using Terminal is not mandatory but is useful for navigating and working with the version control Git. 

```

Comment:
May I suggest using > **_NOTE:_** here instead to make it a block quote and a bit more legible?

Reply:
Worth a try! I'm not sure this will work since this is under the `!!!` section